### PR TITLE
Update ZChannel index.md

### PR DESCRIPTION
**Pull request by counter2015**
fix typo

### CODE REVIEW

File: docs/reference/stream/zchannel/index.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ case class ZPipeline[-R, +E, -In, +Out] (
 
 So we can say that:
 
-- `ZStream[R, E, A]` is a channel that uses `R` as its environment, produce `Chunk[A]` to its output port, can terminate with `Any` success value or can terminate with a failure of type `E`.
+- `ZStream[R, E, A]` is a channel that uses `R` as its environment, produce `Chunk[A]` to its output port, can terminate with a success value of type `A`  or can terminate with a failure of type `E`.
```
Comment (adamgfraser) [inline docs/reference/stream/zchannel/index.md]: This is not correct.

Comment (counter2015) [reply docs/reference/stream/zchannel/index.md]: could you give some adivce on this, how about `can produce elements of type A or can terminate with a failure of type E` ?

Comment (adamgfraser) [reply docs/reference/stream/zchannel/index.md]: `ZStream[R, E, A]` is a channel that requires an environment `R`, emits elements of type `Chunk[A]`, and terminates, if at all, with either an error of type `E` or a done value of type `Any`.

Comment (counter2015) [reply docs/reference/stream/zchannel/index.md]: updated.

**Review by adamgfraser — APPROVED** [review]

**Pull request merged**